unlock-app/src/components/interface/locks/Manage/elements/FilterBar.tsx

```diff
@@ -0,0 +1,170 @@
+import { Button, Input, Select } from '@unlock-protocol/ui'
+import { MdFilterList as FilterIcon } from 'react-icons/md'
+import { BiSearch as SearchIcon } from 'react-icons/bi'
+import { useEffect, useState } from 'react'
+import { MemberFilter } from '~/unlockTypes'
+import useDebounce from '~/hooks/useDebouce'
```

Comment:
btw, there is a debounce hook in `react-use` dependency. 

We can use that one and remove the useDebounce hook in the codebase.

Reply:
+1 to reduce number of dependencies!

Reply:
For now, I merge to avoid waiting for the pipeline and delaying the release.
Opened and assigned this issue to replace it!  https://github.com/unlock-protocol/unlock/issues/9824